### ISSUE TITLE
File parse stops at the first empty line

### ISSUE THREAD

**Issue by MarcoDorantes** [Illyum/l2t#3]
@xxlxii 

If a file has more lines after the first empty line, then those lines are ignored because the DelimiterSeparatedValuesFileParser<T>.Read() method stops at the first empty line.

I have two local branches ready to be published: (1) EmptyLineIssue – a unit test is added to show the evidence of the actual existence of the issue; (2) EmptyLineIssueFixed – the Read method is changed to make the added test pass.

The Pull Request would be created for the (2) EmptyLineIssueFixed branch. Do you agree?

**Comment by MarcoDorantes** [comment]
@xxlxii 

As discussed elsewhere, I have published two branches: EmptyLinesIssue and EmptyLinesIssueFixed.

The EmptyLinesIssue branch contains a unit test that demonstrates the reported issue.

I have created a Pull Request for EmptyLinesIssueFixed branch; which contains a unit test for a file with empty lines and the changes to the DelimiterSeparatedValuesFileParser.Read() method to make that test pass.
